src/term/mod.rs

```diff
@@ -1774,7 +1774,7 @@ impl ansi::Handler for Term {
 
     #[inline]
     fn save_cursor_position(&mut self) {
-        trace!("CursorSave");
+        trace!("save_cursor_position");
```

Comment:
I've mentioned in the previous review that the `CursorSave` should be changed to `Saving cursor`, to be in line with the other comments. What's the reasoning for changing it to `save_cursor_position`? I'd assume it's related to other logs which already existed which mention the calling method, but I think that's not necessary.